flink-tests/src/test/java/org/apache/flink/test/checkpointing/RegionFailoverITCase.java

```diff
@@ -282,7 +301,11 @@ public void initializeState(FunctionInitializationContext context) throws Except
 
 				unionListState = context.getOperatorStateStore().getUnionListState(unionStateDescriptor);
 				Set<Integer> actualIndices = StreamSupport.stream(unionListState.get().spliterator(), false).collect(Collectors.toSet());
-				Assert.assertTrue(CollectionUtils.isEqualCollection(EXPECTED_INDICES, actualIndices));
+				if (getRuntimeContext().getTaskName().contains(SINGLE_REGION_SOURCE_NAME)) {
```

Comment:
Is this change really necessary to resolve FLINK-13452, or can it be in a separate ticket?

Reply:
If we want to let this integration test to include global failover scenario, this is necessary.